packages/a/aws-sdk-cpp/xmake.lua

```diff
@@ -15,6 +15,9 @@ package("aws-sdk-cpp")
     on_load(function (package)
         if package:config("http_client") then
             package:add("deps", "libcurl")
+            if is_plat("macosx") then
```

Comment:
package:is_plat

Reply:
Oh, sorry😅, this is the first time to write xmake-repo, thanks for your help